test/Microsoft.AspNetCore.Mvc.Core.Test/FileResultTest.cs

```diff
@@ -85,6 +86,9 @@ public void Constructor_SetsContentType()
             httpContext
                 .Setup(c => c.RequestServices.GetService(typeof(ILoggerFactory)))
                 .Returns(NullLoggerFactory.Instance);
+            httpContext
```

Comment:
While updating this test, suggest converting it to use a regular `DefaultHttpContext`. Nothing here demands a mock.